src/front-end/typescript/lib/pages/proposal/sprint-with-us/view/tab/code-challenge.tsx

```diff
@@ -170,7 +170,7 @@ const Participants: View<Pick<State, 'proposal'>> = ({ proposal }) => {
             {participants.map((p, i) => {
               const username = p.idpUsername;
               return (
-                <tr>
+                <tr key={username}>
```

Comment:
Not 100% confident about this--normally I'd put the key on the component tag, but there isn't one here

Reply:
Right place for the key; any reason not to use `i`? Wouldn't that be guaranteed to be unique?

```suggestion
                <tr key="participant-{i}">
```

Reply:
I second using `i` here